VK_App/VK_App/ViewControllers/SecondBarVC/protocolConfigurable.swift

```diff
@@ -0,0 +1,12 @@
+// protocolConfigurable.swift
```

Comment:
```suggestion
// NewsConfigurable.swift
```
и название поменяй класса